packages/checkout/widgets-lib/src/widgets/sale/components/FundingRouteExecute/FundingRouteExecute.tsx

```diff
@@ -62,6 +68,7 @@ export function FundingRouteExecute({ fundingRouteStep, onFundingRouteExecuted }
 
   const [swapParams, setSwapParams] = useState<SwapWidgetParams | undefined>(undefined);
   const [bridgeParams, setBridgeParams] = useState<BridgeWidgetParams | undefined>(undefined);
+  const [onRamParams, setOnRampParams] = useState<OnRampWidgetParams | undefined>(undefined);
```

Comment:
small nit: Spelling of `onRamParams` ..
```suggestion
  const [onRampParams, setOnRampParams] = useState<OnRampWidgetParams | undefined>(undefined);
```